controllers/student.go

```diff
@@ -23,6 +23,16 @@ type StudentBlogLinkReqFields struct {
 	BlogLink string `json:"blog_link"`
 }
 
+type StudentDashboard struct {
+	Name     string `json:"name"`
+	Username string `json:"username"`
+	College  string `json:"college"`
+
+	PassedMidEvals bool   `json:"passed_mid_evals"`
+	PassedEndEvals bool   `json:"passed_end_evals"`
+	ProjectsWorked string `json:"projects_worked"`
```

Comment:
Please include all the stats here: CommitCount, PullCount, Lines +/-, LanguagesUsed etc.

Reply:
Also `ProjectsWorked` is just a list of project IDs, we would like to return all the project links instead.
Check [this](https://github.com/kossiitkgp/KWoC-Backend/blob/a0ed5910e8013d7c266031c747de027598de9b46/controllers/stats-student.go#L99) controller for reference.